### ISSUE TITLE
Greenbone Web GUI behind Cloudflare Proxy

### ISSUE THREAD

**Issue by luminous706** [greenbone/gsa#3580]
By putting Greenbone Web GUI behind Cloudflare proxy, I get the following errors:

```
Some cookies are misusing the recommended “SameSite“ attribute 
Cookie “GSAD_SID” does not have a proper “SameSite” attribute value. Soon, cookies without the “SameSite” attribute or with an invalid value will be treated as “Lax”. This means that the cookie will no longer be sent in third-party contexts. If your application depends on this cookie being available in such contexts, please add the “SameSite=None“ attribute to it. To know more about the “SameSite“ attribute, read https://developer.mozilla.org/docs/Web/HTTP/Headers/Set-Cookie/SameSite
```
```
Cookie “GSAD_SID” has been rejected because it is already expired.
```

How to make Greenbone Web GUI work behind Cloudflare proxy?

**Comment by bjoernricks** [comment]
Hi,

as this is a setup question and not a bug report it would be best to raise this question at https://forum.greenbone.net/ instead.

**Comment by cfi-gb** [comment]
This is more related to greenbone/gsad#109 / greenbone/gsad#110 which should be already included in the latest `22.04` stable releases recently published.
